Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
         }
     }
 
-    postBuild {
+    post {
```

Comment:
Unrelated change to make CI pass. Needs https://github.com/stapler/stapler/pull/93 to be merged.